tools/packaging/static-build/qemu/Dockerfile

```diff
@@ -69,11 +69,3 @@ RUN apt-get update && apt-get upgrade -y && \
     GCC_ARCH="${ARCH}" && if [ "${ARCH}" = "ppc64le" ]; then GCC_ARCH="powerpc64le"; fi && \
     if [ "${ARCH}" != "$(uname -m)" ]; then apt-get install --no-install-recommends -y gcc-"${GCC_ARCH}"-linux-gnu; fi && \
     apt-get clean && rm -rf /var/lib/apt/lists/
-
-RUN git clone  https://github.com/axboe/liburing/ ~/liburing && \
-    cd ~/liburing && \
-    git checkout tags/liburing-2.1 && \
-    GCC_ARCH="${ARCH}" && if [ "${ARCH}" = "ppc64le" ]; then GCC_ARCH="powerpc64le"; fi && \
-    if [ "${ARCH}" != "$(uname -m)" ]; then PREFIX="${GCC_ARCH}-linux-gnu"; fi && \
-    ./configure --cc=${GCC_ARCH}-linux-gnu-gcc --cxx=${GCC_ARCH}-linux-gnu-cpp --prefix=/usr/${PREFIX}/ && \
-    make && make install && ldconfig
```

Comment:
@zvonkok - I'm assuming this change is because a liburing pkg is available in jammy, right ? If so, please move that to a separate commit.

Reply:
Alrighty, yeah this commmit grew hysterically.

Reply:
> Alrighty, yeah this commmit grew hysterically.

It is isn't just about that. The changes around liburing should appear in the same patch.